MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -332,6 +332,9 @@ echo "This is an apple."  > ${caption_fn}
 
 export audio_fn="AudioSample.wav"
 wget https://github.com/intel/intel-extension-for-transformers/raw/main/intel_extension_for_transformers/neural_chat/assets/audio/sample.wav -O ${audio_fn}
+
+export audio_mp3_fn="sample_audio.mp3"
+wget wget https://audio-samples.github.io/samples/mp3/blizzard_biased/sample-0.mp3 -O ${audio_mp3_fn}
```

Comment:
```suggestion
wget https://audio-samples.github.io/samples/mp3/blizzard_biased/sample-0.mp3 -O ${audio_mp3_fn}
```

Reply:
Also, not sure on the licensing of this blizzard dataset. It might be safer to use another ITREX sample. For example:
```
wget https://github.com/intel/intel-extension-for-transformers/raw/refs/tags/v1.5/intel_extension_for_transformers/neural_chat/ui/customized/talkingbot/src/lib/components/talkbot/assets/mid-age-man.mp3
```

Reply:
Another idea is that instead of adding another audio file download to this example (the list of downloads is getting long), maybe the documentation should list what types of files are supported. For example, even for image ingestion, we support png, gif, and jpg, but I don't see that listed in the documentation. 🤔